### PR TITLE
feat(cubejs-api-gateway): Add description to V1CubeMeta, V1CubeMetaDimension and V1CubeMetaMeasure in OpenAPI

### PR DESCRIPTION
**Pull request by mcheshkov**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**

Add new field `description` in OpenAPI schema

### CODE REVIEW

File: packages/cubejs-api-gateway/test/index.test.ts [hunk 1]
```diff
@@ -579,7 +579,10 @@ describe('API Gateway', () => {
       .expect(200);
     expect(res.body).toHaveProperty('cubes');
     expect(res.body.cubes[0]?.name).toBe('Foo');
+    expect(res.body.cubes[0]?.description).toBe('cube from compilerApi mock');
```
Comment (KSDaemon) [inline packages/cubejs-api-gateway/test/index.test.ts:582]: If you're here, can you fix the warning above ^^^ too please?

**Review by KSDaemon — APPROVED** [review]
👍🏻 LGTM!

File: packages/cubejs-api-gateway/test/index.test.ts (hunk 1 above)
Comment (mcheshkov) [reply packages/cubejs-api-gateway/test/index.test.ts:581]: Done

File: rust/cubesql/cubeclient/src/models/v1_cube_meta.rs [hunk 2]
```diff
@@ -30,15 +32,15 @@ impl V1CubeMeta {
...
     ) -> V1CubeMeta {
         V1CubeMeta {
             name,
             title: None,
+            description: None,
             measures,
             dimensions,
             segments,
-            joins,
+            joins: None,
```
Comment (ovr) [inline rust/cubesql/cubeclient/src/models/v1_cube_meta.rs:43]: @mcheshkov m? did we remove it?

Comment (mcheshkov) [reply rust/cubesql/cubeclient/src/models/v1_cube_meta.rs:43]: It's not marked as `required` in schema, so `openapi-generator` dropped in from `new`. I don't think we are using `V1CubeMeta::new` anyway.

https://github.com/cube-js/cube/blob/05ea7e25014b7ac72e32fa0391ce3ea99ab3e01f/packages/cubejs-api-gateway/openspec.yml#L125-L129

**Review by paveltiunov — APPROVED** [review]

**Pull request merged**